pyaptamer/datasets/_loaders/load_aptamer_interactions.py

```diff
@@ -0,0 +1,235 @@
+__author__ = "Satarupa22-SD"
+__all__ = ["load_aptadb", "load_aptamer_interactions", "load_interactions"]
+
+from pathlib import Path
+
+import pandas as pd
+
+
+def _download_dataset(
+    dataset_name: str, target_dir: Path, force_download: bool = False
+) -> None:
+    """Download a Kaggle dataset to the specified directory and unzip it.
+
+    This is a private helper function used internally by the module.
```

Comment:
An underscore before the function name indicates it is a pvt function, this line can be deleted.